types/walk-asyncgen/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "es2018"
```

Comment:
```suggestion
            "ES2018.AsyncGenerator"
```

Reply:
This is what I had originally set it to, but my IDE (webstorm) complained about "Regex failed" so I was confused.

Reply:
that's ok to go with full import, but if that is still a problem, let me know

Reply:
I will.